manifests/p/ppy/osu/2024.1224.1/ppy.osu.locale.en-US.yaml

```diff
@@ -0,0 +1,81 @@
+  - Change slider drop penalty to use actual number of difficult sliders, fix slider drop penalty being too lenient (ppy/osu#31055 by @stanriders)
+  - Use lastAngle when nerfing repeated angles on acute bonus (ppy/osu#31245 by @tsunyoku)
+  Editor
+  Change beat snap divisor adjust defaults to be Ctrl+Scroll instead of Ctrl+Shift+Scroll (ppy/osu#31147 by @peppy)
+  Smaller changes
+  - Fix being able to click through slider tail drag handles (ppy/osu#31184 by @peppy)
+  Framework
+  - Fix BindableList methods crashing with non-IList collections (ppy/osu-framework#6466 by @smoogipoo)
+  - Move "global" scroll-adjusts-volume to a per-screen component-based implementation (ppy/osu#31146 by @peppy)
+  - Implement getting OpenGL iOS main framebuffer on SDL3 (ppy/osu-framework#6468 by @Susko3)
+  - Fix typo in main menu tip (ppy/osu#31224 by @frenzibyte)
+  - Use Accelerate framework for texture image processing on iOS and fix premultiplication issues (ppy/osu-framework#6470 by @frenzibyte)
+  - Shave allocation overheads in recent iOS texture loading code (ppy/osu-framework#6471 by @frenzibyte)
+  Gameplay
+  Skip the pause cooldown when in intro / break time (ppy/osu#31143 by @peppy)
+  Implement skinnable mod display (ppy/osu#30993 by @Tom94)
+  Gameplay (osu!)
+  Fix some hitcircle sprites missing in certain skins since last release (ppy/osu#31254 by @peppy)
+  Gameplay (osu!mania)
+  - Fix "spinner" conversion for mania-specific beatmaps (ppy/osu#30984 by @smoogipoo)
+  Main Menu
+  - Load seasonal backgrounds without requiring being logged in (ppy/osu#31198 by @peppy)
+      - No reason not to.
+  - Add christmas / seasonal mode (ppy/osu#31206 by @peppy)
+  - Slightly reduce background brightness at main menu when seasonal lighting is active (ppy/osu#31258 by @peppy)
+  - Fix adjusting volume using scroll wheel not working during intro (ppy/osu#31259 by @peppy)
+  Platform
+  Fix logo animation not rendering correctly on iOS (ppy/osu-resources#351 by @frenzibyte)
+  Save "featured artists" filter to user settings (ppy/osu#31238 by @frenzibyte)
+  Also sets it to always on for iOS in preparation for a larger scale release.Smaller changes
+  - Update game structure with the returned iOS application delegates (ppy/osu#31152 by @frenzibyte)
+  - Fix hardware cursor showing on iPadOS when it shouldn't (ppy/osu#31226 by @frenzibyte)
+  Settings
+  Add ability to rename the current skin (ppy/osu#31220 by @normalid-awa)
+  In settings for now, very temporary.Song Select
+  Implement ability to mark beatmap as played (ppy/osu#31138 by @bdach)
+  Would you believe it that this button that has been there for literal years never did anything? Also available via context menu.Tournament
+  Hide !mp commands from tournament streaming chat (ppy/osu#31150 by @peppy)
+  Full Changelog: https://github.com/ppy/osu/compare/2024.1219.2...2024.1224.1
+ReleaseNotesUrl: https://github.com/ppy/osu/releases/tag/2024.1224.1

```

Comment:
```suggestion
ReleaseNotes: |
  Difficulty Calculation
    ✓ Refactor TaikoDifficultyCalculator and add DifficultStrain attributes (osu#31191)by Lawtrohux
    ✓ Increase accscalingshift and include countok in hit proportion (osu#31195)by Lawtrohux
    ✓ Remove Conditional SkillRating nerf to Converts (osu#31196)by Lawtrohux
    ✓ Simplify osu! high-bpm acute angle jumps bonus (osu#30902)by stanriders
    + Implement Reading Skill into osu!taiko (osu#31208)by Lawtrohux
    ✓ Change slider drop penalty to use actual number of difficult sliders, fix slider drop penalty being too lenient (osu#31055)by stanriders
    ✓ Use lastAngle when nerfing repeated angles on acute bonus (osu#31245)by tsunyoku
  Editor
    ✓ Fix being able to click through slider tail drag handles (osu#31184)by peppy
    ✓ Change beat snap divisor adjust defaults to be Ctrl+Scroll instead of Ctrl+Shift+Scroll (osu#31147)by peppy
  Framework
    ✓ Fix BindableList methods crashing with non-IList collections (osu-framework#6466)by smoogipoo
    ✓ Move "global" scroll-adjusts-volume to a per-screen component-based implementation (osu#31146)by peppy
    + Implement getting OpenGL iOS main framebuffer on SDL3 (osu-framework#6468)by Susko3
    ✓ Fix typo in main menu tip (osu#31224)by frenzibyte
    ✓ Use Accelerate framework for texture image processing on iOS and fix premultiplication issues (osu-framework#6470)by frenzibyte
    ✓ Shave allocation overheads in recent iOS texture loading code (osu-framework#6471)by frenzibyte
  Gameplay
    ✓ Skip the pause cooldown when in intro / break time (osu#31143)by peppy
    + Implement skinnable mod display (osu#30993)by Tom94
    Gameplay (osu!)
      ✓ Fix some hitcircle sprites missing in certain skins since last release (osu#31254)by peppy
    Gameplay (osu!mania)
      ✓ Fix "spinner" conversion for mania-specific beatmaps (osu#30984)by smoogipoo
  Main Menu
    ✓ Load seasonal backgrounds without requiring being logged in (osu#31198)by peppy
      No reason not to.
    + Add christmas / seasonal mode (osu#31206)by peppy
    ✓ Slightly reduce background brightness at main menu when seasonal lighting is active (osu#31258)by peppy
    ✓ Fix adjusting volume using scroll wheel not working during intro (osu#31259)by peppy
  Platform
    ✓ Update game structure with the returned iOS application delegates (osu#31152)by frenzibyte
    ✓ Fix logo animation not rendering correctly on iOS (osu-resources#351)by frenzibyte
    ✓ Fix hardware cursor showing on iPadOS when it shouldn't (osu#31226)by frenzibyte
  Settings
    ✓ Save "featured artists" filter to user settings (osu#31238)by frenzibyte
      Also sets it to always on for iOS in preparation for a larger scale release.
    + Add ability to rename the current skin (osu#31220)by normalid-awa
      In settings for now, very temporary.
  Song Select
    + Implement ability to mark beatmap as played (osu#31138)by bdach
      Would you believe it that this button that has been there for literal years never did anything? Also available via context menu.
  Tournament
    ✓ Hide !mp commands from tournament streaming chat (osu#31150)by peppy
  Full Changelog: https://github.com/ppy/osu/compare/2024.1219.2...2024.1224.1
ReleaseNotesUrl: https://osu.ppy.sh/home/changelog/lazer/2024.1224.1
```